source/features/parse-backticks.tsx

```diff
@@ -24,7 +24,9 @@ function init(): void {
 		'.Box-row.js-navigation-item a.link-gray-dark', // `isGlobalDiscussionList`
 		'[data-channel^="check_suites"] a', // `isActions`
 		'.repository-content .pr-toolbar h2', // `isActions` run
-		'#wiki-wrapper h1' // `isWiki`
+		'#wiki-wrapper h1', // `isWiki`
+		'a[data-hydro-click*="issue_cross_references.click"]', // `isIssue` Linked pull requests
```

Comment:
Wrong/incomplete/sloppy as usual, even after 3 requests. This also works on isPRConversation / Linked issues.

See https://github.com/sindresorhus/refined-github/pull/3022#issuecomment-617270396